views/models/ChartModel/index.js

```diff
@@ -1,5 +1,15 @@
+import { SAFE_URL_PATTERN, DATA_URL_PATTERN } from '../../../constants'
+
 const ChartModel = {
   isOwnedByUser: (chart, user) => chart.owner === user.uid,
+  sanitiazeProfileImageUrl: (url = 'about:blank') => {
+    url = String(url)
+    if (url === 'null' || url.length === 0 || url === 'about:blank')
+      return 'about:blank'
+    if (url.match(SAFE_URL_PATTERN) || url.match(DATA_URL_PATTERN)) return url
+
+    return `unsafe:${url}`
```

Comment:
should this throw an error instead of returning an unsafe URL?

Reply:
I think returning something is actually better, but maybe we can drop in a big "X" or "Unsafe URL". We don't want the app to choke for everybody if one person sets a bad profile picture

Reply:
Example return value: `data:,Unsafe%20Image`

Reply:
[05e9f09](https://github.com/AMP-SCZ/dpdash/pull/530/commits/05e9f090fc738b3090970d1e415c0b1966628835)